scripts/INSTALL_prerequisites_with_apt-get.sh

```diff
@@ -21,10 +21,7 @@ $SUDO apt-get install -y --no-install-recommends swig
 
 # TODO replace with pip install
 echo "Installing python libraries etc"
-$SUDO apt-get install -y --no-install-recommends  python-scipy python-docopt  python-numpy python-h5py python-matplotlib python-libxml2 python-psutil spyder python-tk python-nose
+$SUDO apt-get install -y --no-install-recommends  python-scipy python-docopt  python-numpy python-h5py python-matplotlib python-libxml2 python-psutil python-tk python-nose
```

Comment:
please add python-pip here

Reply:
I also add firefox

Reply:
yes, but not in this file, but in Vagrantfile, which seemed fine to us. could leave the pip in Vagrantfile as well if you prefer actually.

Reply:
Could you tell me your rationale? So maybe I will do it right, next time.

Reply:
Actually, reading the initial comment it seems to describe the rationale. However, as far as I understand, @bathomas is not exactly using  `INSTALL_prerequisites_with_apt-get.sh` and `UPDATE.sh`. He has a [`provision.sh`](https://github.com/bathomas/stir-azure/blob/master/provision.sh) which does a mixture.